src/data/contributors/siddhant.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Siddhant Khobragade",
+  "skills": ["Java", "Android Front-end Development"],
+  "cause": "Community empowerment",
+  "image": "Avatar5",
+  "github": "https://github.com/siddhantdrk",
+  "twitter": "Nill",
```

Comment:
You can keep the string empty but putting ```nill``` is not allowed.

Reply:
I made the changes and updated respectively